mreg/models.py

```diff
@@ -605,7 +605,7 @@ class ModelChangeLog(models.Model):
     table_row = models.BigIntegerField()
     data = models.TextField()
     action = models.CharField(max_length=16)  # saved or deleted
-    timestamp = models.DateTimeField()
+    timestamp = models.DateTimeField(default=timezone.now)
```

Comment:
Even better to add `auto_now_add=True`.

Reply:
Also no migration-file added. `python manage.py makemigrations`.

Reply:
Enig at auto_now_add er bedre.  Endret.  Kan du fikse migreringsscript når endringen tas inn.  Best å utsette til siste sekund, i tilfelle andre databaseendringen kommer inn før denne.